### PR TITLE
py-brotlicffi: new port (version 1.0.9.2)

### PR DESCRIPTION
**Pull request by catap**
#### Description

`py-brotlipy` was renamed to `py-brotlicffi` for a while, and I've made the same at MacPorts.

I've also added `py311` subport for `py-brotli`.

###### Type(s)

- [x] enhancement

###### Tested on

macOS 12.6 21G115 x86_64
Xcode 14.1 14B47b

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint --nitpick`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Review by raimue — APPROVED** [review]
Looks good to me! Thank you for the pull request. :+1:

**Comment by catap** [comment]
@raimue may I ask you to merge it when? ;)

File: python/py-brotlicffi/Portfile [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+} else {
+    set pybrotlipy_name py-brotlipy
+}
+
+#------------------------------------------------------------------------------
+# NOTE: Remove this logic - along with declaration of 'pybrotlipy_name', above,
+#   when obsolete port 'py-pybrotlipy' is ultimately deleted.
+#------------------------------------------------------------------------------
+pre-activate {
+    # Deactivate obsolete port 'py-crypto', if installed
```
Comment (raimue) [inline python/py-brotlicffi/Portfile:40]: Should be 'py-brotlipy', right?

Comment (catap) [reply python/py-brotlicffi/Portfile:40]: thanks to catching this copy and paste issue!

**Review by raimue — APPROVED** [review]

**Comment by catap** [comment]
Thanks!

**Pull request merged**